src/popup/popup.xhtml

```diff
@@ -44,13 +44,13 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 				<div class="panel-section">
 					<i class="eb-icon-placeholder"/>
 					<i class="panel-list-item text" data-i18n="popup_edit"/>
-					<div class="panel-list-item icon" data-ipc-message="editCut" data-remain-open="true">
+					<div class="panel-list-item icon" data-ipc-message="editCut">
```

Comment:
`data-remain-open="true"` is needed here to accurately implement how the Application Menu worked in Firefox 4-28 and CTR for FF 29-56.